sormas-rest/src/main/webapp/WEB-INF/web.xml

```diff
@@ -40,6 +43,7 @@
 			<role-name>USER</role-name>
 			<role-name>REST_EXTERNAL_VISITS_USER</role-name>
 			<role-name>REST_USER</role-name>
+			<role-name>SORMAS_TO_SORMAS_CLIENT</role-name>
```

Comment:
separate security-constraint for SORMAS_TO_SORMAS_CLIENT that protects /sormasToSormas ?

Reply:
display-name should be e.g. "secure sormasToSormas".
web-resource-name seems to be optional, so I guess it can be removed.

Reply:
> display-name should be e.g. "secure sormasToSormas".
> web-resource-name seems to be optional, so I guess it can be removed.